.env

```diff
@@ -130,3 +130,7 @@ GOACCESS_KEEP_LAST_DAYS=60
 # SFTP
 SFTP_IMAGE=atmoz/sftp:alpine-3.7
 SFTP_PORT=10022
+
+# Webcam
+WEBCAM_KEEP_DAYS=7
```

Comment:
Let's add an inline explanation for these variables.

```suggestion
# How many days (including the current) should be mirrored.
WEBCAM_KEEP_DAYS=7
```

Reply:
Added